### PR TITLE
fix lint warnings

### PR DESCRIPTION
**Pull request by sona-podholova**
# TL;DR

fixing lint warnings

# Proof
image

# Merge request checklist

Please check if your merge request fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been reviewed and added/updated if needed (for bug fixes/features)
- [x] Build (`yarn build`) was run locally and any changes were pushed for both API and clients
- [x] API (`yarn api:dev`) runs locally and any fixes were made for failures
- [x] Admin Client (`cd clients/admin && yarn start`) runs locally and any fixes were made for failures

# Merge request type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no APIoduce  changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

# Before this merge request
Lint warnings

# How you fixed what was wrong
removed unused stuff mainly

### CODE REVIEW

**Review by davidpodhola — CHANGES_REQUESTED** [review]
there are unfortunately still 3 warnings

**Review by davidpodhola — APPROVED** [review]
